admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -154,6 +154,95 @@ In openSUSE, modify the ``/usr/sbin/start_apache2`` file::
 
 Restart Apache, open your ownCloud Admin page and start creating SMB/CIFS mounts.
 
+Create Shares Programmatically
+------------------------------
+
+If you need to create new shares using command-line scripts, there are two available option.
+
+- `occ files_external:create`_
+- `occ files_external:import`_
+
+occ files_external:create
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This command provides for the creation of both personal (for a specific user) and general shares.
+The command’s configuration options can be provided either as individual arguments or collectively, as a JSON object.
+For more information about the command, refer to the :ref:`the occ documentation <files_external_create_label>`.
+
+**Personal share**
+
+::
+
+    occ files_external:create /my_share_name windows_network_drive \
+        password::logincredentials \
+        --config={host=127.0.0.1, share='home', root='$user', domain='owncloud.local'} \
+        --user someuser
+
+::
+ 
+    occ files_external:create /my_share_name windows_network_drive \
```

Comment:
For all occ commands
``occ files_external`` --> ``sudo -u www-data php occ files_external``